cmd/sync/sync.go

```diff
@@ -36,9 +36,10 @@ var (
 type sync struct {
 	azs  azureclient.AccountsClient
 	blob azureclientstorage.Blob
+	log  *logrus.Entry
 }
 
-func (s *sync) init(ctx context.Context) error {
+func (s *sync) init(ctx context.Context, entry *logrus.Entry) error {
```

Comment:
`, log *logrus.Entry` would be preferred

Reply:
Updated.